pkg/maps/loader.go

```diff
@@ -456,9 +456,12 @@ func (m *BpfMap) BulkDeleteMapEntry(keyvalue map[uintptr]uintptr) error {
 	return nil
 }
 
-func (m *BpfMap) BulkUpdateMapEntry(keyvalue map[uintptr]uintptr) error {
+func (m *BpfMap) BulkUpdateMapEntry(keyvalue map[string]uintptr) error {
 	for k, v := range keyvalue {
-		err := m.UpdateMapEntry(k, v)
+		keyByte := []byte(k)
+		log.Infof("Converted string to bytearray %v", keyByte)
```

Comment:
Log will probably be too verbose and byte value is tough to decode anyways...so might not be helpful to even print this out...

Reply:
ok, this kind of helped debug current issue..